services/galley/src/Galley/API/Teams/Features.hs

```diff
@@ -590,7 +590,8 @@ getSelfDeletingMessagesInternal = \case
           Public.PaymentUnlocked
       (Just Public.PaymentUnlocked, Nothing) ->
         Public.defaultSelfDeletingMessagesStatus {Public.tfwcapsPaymentStatus = Public.PaymentUnlocked}
-      _ -> Public.defaultSelfDeletingMessagesStatus
+      (Just Public.PaymentLocked, _) -> Public.defaultSelfDeletingMessagesStatus
+      (Nothing, _) -> Public.defaultSelfDeletingMessagesStatus
```

Comment:
right, i missed the `Nothing`.  makes it slightly less pretty, but i still prefer the way it reads now.  thanks!